Source/WebCore/rendering/style/RenderStyleConstants.h

```diff
@@ -1198,6 +1198,13 @@ enum class ContentVisibility : uint8_t {
 };
 
 // Stored as unsigned : 2 in StyleRareNonInheritedData
```

Comment:
Probably want to keep a copy of this comment on `BlockStepInsert`.

Reply:
I'll drop the comment, none of the other enums have it, and it's not super useful